testkit-backend/src/main/java/neo4j/org/testkit/backend/messages/requests/StartTest.java

```diff
@@ -81,9 +82,9 @@ public class StartTest implements TestkitRequest {
                 "^.*\\.TestOptimizations\\.test_uses_implicit_default_arguments_multi_query$", skipMessage);
         COMMON_SKIP_PATTERN_TO_REASON.put(
                 "^.*\\.TestOptimizations\\.test_uses_implicit_default_arguments_multi_query_nested$", skipMessage);
-        skipMessage = "Additional type support is needed";
         COMMON_SKIP_PATTERN_TO_REASON.put(
-                "^neo4j\\.datatypes\\.test_temporal_types\\.TestDataTypes\\..*$", skipMessage);
+                "^.*\\.test_unknown_then_known_zoned_date_time(_patched)?$",
```

Comment:
https://github.com/neo4j-drivers/testkit/pull/488/files#r913885423

Reply:
Can this be merged regardless of this discussion? We might decide to remove the test later down the road if we don't want our drivers to behave like this. For now, skipping it seems ok to me regardless of the decision we'll make.

Reply:
Sure.